cinema_service/settings.py

```diff
@@ -20,12 +22,9 @@
 # See https://docs.djangoproject.com/en/4.0/howto/deployment/checklist/
 
 # SECURITY WARNING: keep the secret key used in production secret!
-SECRET_KEY = (
-    "django-insecure-6vubhk2$++agnctay_4pxy_8cq)mosmn(*-#2b^v4cgsh-^!i3"
-)
+SECRET_KEY = os.getenv("SECRET_KEY", "your-default-secret-key")
```

Comment:
It's generally not advisable to include a default value for your SECRET_KEY in your code. If the environment variable isn't set, your application should not run. This is to prevent your application from running in an insecure state.